### PR TITLE
Adds close discussion button and behaviour

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

File: app/assets/javascripts/app/discussions/discussion.js [hunk 1]
```diff
@@ -19,6 +20,11 @@
       .find(discussionId)
       .then(updateDiscussion);
 
+    function closeDiscussion(discussion) {
+      discussion.open = !discussion.open;
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/discussion.js]: This doesn't sound like a toggle, why aren't you assigning false here?

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/discussion.js]: Because this way it can be used to reopen the discussion too

Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/discussion.js]: Read the name of the function

Comment (joaojusto) [inline app/assets/javascripts/app/discussions/discussion.js]: Ok I'll change the name

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**